src/Homework2.java

```diff
@@ -0,0 +1,21 @@
+public class Homework2 {
+    public static boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
+        if ((a ^ b) && (c ^ d) || (a ^ c) && (b ^ d)) {
+            return true;
+        } else return false;
+    }
+
+    public static int leapYearCount(int year) {
+        return year = year / 4 - year / 100 + year / 400;
+    }
+
+    final static double fault = 0.0001;
```

Comment:
add access modifier and read about constant names
https://mate-academy.github.io/style-guides/java/java.html#s5-naming